docker/build.sh

```diff
@@ -48,11 +48,19 @@ with_timeout()
 # Build bpftrace
 mkdir -p "$1"
 cd "$1"
-cmake -DCMAKE_BUILD_TYPE="$2" -DWARNINGS_AS_ERRORS:BOOL=$WARNINGS_AS_ERRORS \
-      -DSTATIC_LINKING:BOOL=$STATIC_LINKING -DSTATIC_LIBC:BOOL=$STATIC_LIBC \
-      -DEMBED_LLVM:BOOL=$EMBED_LLVM -DEMBED_CLANG:BOOL=$EMBED_CLANG \
-      -DEMBED_LIBCLANG_ONLY:BOOL=$EMBED_LIBCLANG_ONLY \
-      -DLLVM_VERSION=$LLVM_VERSION  ../
+if [ ${DISTRO} = "xenial" ]; then
```

Comment:
I'd suggest that we just put the extra CXX flags in a variable, which we then conditionally set (but unconditionally interpolate). I did something similar on my android WIP branch: https://github.com/iovisor/bpftrace/pull/1095/files#diff-c0d65d2fb9fff450085d1f1a44ca5bb1R76

Reply:
did exactly this and thought I was crazy because it made cmake very unhappy. let me give it another shot.